### PR TITLE
Fix content layout shift with mod-actions-menu-btn

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fixes a content layout shift in the show view action panel for trusted shield. This re-uses the `trusted-visible-block` which is designed for this type of functionality. 🙂

## Related Tickets & Documents

- Closes https://github.com/forem/forem/issues/19990

## QA Instructions, Screenshots, Recordings

Get into any "trusted" state. If you sign in as admin on Uffizzi, you should be able to see this.

(Email: `admin@forem.local`, password: `password`)

### UI accessibility checklist

This should improve a11y as a side effect.

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [x] Yes

## [optional] What gif best describes this PR or how it makes you feel?

satisfying

### CODE REVIEW

File: app/views/moderations/mod.html.erb [hunk 1]
```diff
@@ -71,7 +71,7 @@
       <a href="/resource_admin/articles/<%= @moderatable.id %>" data-no-instant><%= t("views.moderations.actions.edit.resource_admin") %></a> |
       <a href="<%= admin_article_path(@moderatable.id) %>" data-no-instant><%= t("views.moderations.actions.edit.article_admin") %></a></h3>
   <% elsif current_user.super_admin? && @moderatable.class.name == "Comment" %>
-    <h3> <a href="/admin/comments/<%= @moderatable.id %>" data-no-instant><%= t("views.moderations.actions.edit.comment_admin") %></a> |
+    <h3> <a href="/admin/content_manager/comments/<%= @moderatable.id %>" data-no-instant><%= t("views.moderations.actions.edit.comment_admin") %></a> |
```
Comment (benhalpern) [inline app/views/moderations/mod.html.erb:74]: This is a small fix to a URL that had been broken for a while within comment mod change, not directly related to this fix but along for the ride.

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -98,7 +98,7 @@ body.mod-status-true .mod-icon-visible-block {
 }
 
 body.trusted-status-true .trusted-visible-block {
-  display: flex !important;
+  display: block !important;
```
Comment (filleduchaos) [inline app/assets/stylesheets/scaffolds.scss:101]: I don't _think_ this causes any issues, but in the other place that this is used (the nav menu dropdown) the other links use flex layout?

**Review by filleduchaos — APPROVED** [review]

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

File: app/assets/stylesheets/scaffolds.scss (hunk 2 above)
Comment (benhalpern) [reply app/assets/stylesheets/scaffolds.scss:101]: Oh yeah, I meant to comment on that: Yes @filleduchaos this does not cause any issues. The only other place it uses it is indifferent to block vs flex.

**Pull request merged**